chain/consensus/mir/validator/membership.go

```diff
@@ -32,3 +36,27 @@ type EnvMembership string
 func (e EnvMembership) GetValidatorSet() (*Set, error) {
 	return NewValidatorSetFromEnv(string(e))
 }
+
+// -----
+
+const JSONRPCServerURL = "http://127.0.0.1:3030"
```

Comment:
We should figure out a way to pass this as a config parameter when we start the daemon. We should allow a way to pass the endpoint for the ipc-agent related to the lotus instance.